src/zope/traversing/api.py

```diff
@@ -92,21 +70,8 @@ def traverse(object, path, default=_marker, request=None):
 
 
 def traverseName(obj, name, default=_marker, traversable=None, request=None):
-    """Traverse a single step 'name' relative to the given object.
-
-    'name' must be a string. '.' and '..' are treated specially, as well as
-    names starting with '@' or '+'. Otherwise 'name' will be treated as a
-    single path segment.
-
-    You can explicitly pass in an ITraversable as the 'traversable'
-    argument. If you do not, the given object will be adapted to ITraversable.
-
-    'request' is passed in when traversing from presentation code. This
-    allows paths like @@foo to work.
-
-    Raises LocationError if path cannot be found and 'default' was
-    not provided.
-
+    """
+    Traverse a single step 'name' relative to the given object.
```

Comment:
~~It looks like useful description is being lost.~~

This seemed peculiar before I saw the interface `__doc__` copying at the end of the module.